packages/agents/chain-abstraction/src/helpers/swapdata.ts

```diff
@@ -11,6 +11,15 @@ export type OriginSwapDataCallbackArgs = {
   toAsset: string;
   amountIn: string;
   fromAddress: string;
+  config?:
+    | {
+        customURL: string;
+        apiKey?: string;
+      }
+    | {
+        customURL?: undefined;
+        apiKey: string;
+      };
```

Comment:
I guess we can make it better like 

```suggestion
  config?:
   {
        customURL?: string;
        apiKey?: string;
   }
```